tests/tools/test_theoretical_simulation.py

```diff
@@ -0,0 +1,19 @@
+import numpy as np
+
+from mpqp import QCircuit
+from mpqp.core.instruction.measurement import BasisMeasure
+from mpqp.gates import *
+from mpqp.tools.theoretical_simulation import theoretical_probs
+
+
+def test_simulation():
```

Comment:
What about testing the theoretical simulation in noisy case without comparing to the output of an SDK, but compared to a density matrix we computed analytically (by hand or using Maple for instance ?)

Reply:
this will come in a future PR